content/200-orm/100-prisma-schema/20-data-model/50-database-mapping.mdx

```diff
@@ -94,9 +94,9 @@ Some use cases for explicitly named constraints include:
 - Company policy
 - Conventions of other tools
 
-#### Prisma's default naming conventions for indexes and constraints
+#### Prisma ORM's default naming conventions for indexes and constraints
 
-Prisma naming convention was chosen to align with PostgreSQL since it is deterministic. It also helps to maximize the amount of times where names do not need to be rendered because many databases out there they already align with the convention.
+Prisma ORM naming convention was chosen to align with PostgreSQL since it is deterministic. It also helps to maximize the amount of times where names do not need to be rendered because many databases out there they already align with the convention.
 
 We always use the database names of entities when generating the default names. So if a model is remapped to a different name in the data model, the default name generation will still take the name of the table in the database as input. The same is true for fields and columns.
```

Comment:
Note to myself: This belong _after_ the table, and probably needs to be clarified to make sense. I needed to read it multiple times to understand what it is talking about.

Reply:
```suggestion
Prisma ORM always uses the database names of entities when generating the default index and constraint names. If a model is remapped to a different name in the data model via `@@map` or `@map`, the default name generation will still take the name of the _table_ in the database as input. The same is true for fields and _columns_.
```